### PR TITLE
feat(obstacle_avoidance_planner): parameterize bounds search widths

### PR DESCRIPTION
**Pull request by takayuki5168**
## Description

Just set bounds_search_widths as a ros parameter, which will be different depending on the vehicle size

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by kosuke55 — APPROVED** [review]
LGTM

**Comment by kosuke55** [comment]
I also confirmed that the parameters are reflected with https://github.com/tier4/autoware_launch/pull/295
` ros2 param dump  --print /planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner`

File: launch/tier4_planning_launch/config/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/obstacle_avoidance_planner.param.yaml [hunk 1]
```diff
@@ -123,6 +123,8 @@
           eps_rel: 1.0e-10 # eps rel when solving OSQP
 
       mpt:
+        bounds_search_widths: [0.45, 0.15, 0.05, 0.01]
```
Comment (yukkysaito) [inline launch/tier4_planning_launch/config/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/obstacle_avoidance_planner.param.yaml:126]: Can you describe the meaning of the parameters in the readme?

Comment (takayuki5168) [reply launch/tier4_planning_launch/config/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/obstacle_avoidance_planner.param.yaml:126]: https://github.com/autowarefoundation/autoware.universe/pull/807/commits/73c6a9d6c0570bc96315aa6fe0947b417a9dd6c9
DONE

**Review by kosuke55 — APPROVED** [review]

**Pull request merged**